jgiven-examples/src/test/java/com/tngtech/jgiven/examples/pancakes/test/ExtendedVocabularyPanCakeTest.java

```diff
@@ -0,0 +1,93 @@
+package com.tngtech.jgiven.examples.pancakes.test;
+
+import com.tngtech.jgiven.Stage;
+import com.tngtech.jgiven.annotation.FillerWord;
+import com.tngtech.jgiven.annotation.ProvidedScenarioState;
+import com.tngtech.jgiven.annotation.Punctuation;
+import com.tngtech.jgiven.examples.pancakes.app.Cook;
+import com.tngtech.jgiven.examples.pancakes.app.PanCakeCook;
+import com.tngtech.jgiven.examples.pancakes.test.steps.ThenMeal;
+import com.tngtech.jgiven.examples.pancakes.test.steps.WhenCook;
+import com.tngtech.jgiven.junit.ScenarioTest;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.tngtech.jgiven.examples.pancakes.test.ExtendedVocabularyPanCakeTest.GivenIngredients;
+
+
+public class ExtendedVocabularyPanCakeTest extends ScenarioTest<GivenIngredients, WhenCook, ThenMeal> {
+
+    @ProvidedScenarioState
+    private final Cook cook = new PanCakeCook();
+
+    @Test
+    public void a_pancake_can_be_fried_out_of_an_egg_milk_and_flour() {
+
+        given() .ingredients().comma() .consisting_of()
+            .asterisk() .an() .egg()
```

Comment:
in this case, I'd argue that `asterisk` is functionally closer to a word than to punctuation. But I get now, why you implemented it the way you did. I'm just not completely satisfied...

Reply:
Ok, agreed. Maybe I overdid this one. I'll wind back the use of punctuation a little.